community/Hexo.gitignore

```diff
@@ -0,0 +1,19 @@
+# gitignore template for Hexo sites
+# website: https://hexo.io/
+# Recommended: Node.gitignore
+
+# Ignore generated files and directories
+.tmp*
+dist/
+public/
+coverage/
+
+# Ignore temp files and lock files
+tmp/
+yarn.lock
+package-lock.json
+pnpm-lock.yaml
```

Comment:
Generally, package lock files should be included in source control. You can see that yarn's guidance on this [here](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored).

If for some reason this is different for hexo, please let me know (references are helpful here)! Otherwise, could you remove these lock files from the template?

Reply:
Apologize for not providing a full reference, in fact I referenced [.gitignore at hexojs/site](https://github.com/hexojs/site/blob/master/.gitignore) as well, which include the lock files

Reply:
I read the guidance of yarn carefully and I agree with removing them from the template